controller/cmd/service-mirror/metrics.go

```diff
@@ -67,7 +67,7 @@ func newProbeMetricVecs() probeMetricVecs {
 	latencies := promauto.NewHistogramVec(
 		prometheus.HistogramOpts{
 			Name: "gateway_probe_latency_ms",
-			Help: "A histogram of latencies to a remote gateway.",
+			Help: "A histogram of latencies to a target gateway.",
```

Comment:
Consider "to a target cluster gateway" or "to a gateway" or "to a gateway in a target cluster"